### ISSUE TITLE
\\sshfs\<USER_NAME>@<DOMAIN>!<PORT>\ does not work

### ISSUE THREAD

**Issue by tgckpg** [billziss-gh/sshfs-win#8]
Connecting to a different port will continuously prompt for passwords.
`systeminfo`
```
OS Name:                   Microsoft Windows 10 Pro
OS Version:                10.0.14393 N/A Build 14393

Network Card(s):           3 NIC(s) Installed.
                           [01]: Realtek PCIe GBE Family Controller
                                 Connection Name: Ethernet
                                 DHCP Enabled:    Yes
                                 DHCP Server:     N/A
                                 IP address(es)
                           [02]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (vlan)
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 192.168.0.222
                                 [02]: fe80::aca2:c28e:2000:f1a9
                           [03]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (Internal Ethernet Port Windows Phone Emulator Internal Switch)
```

**Comment by billziss-gh** [comment]
Thanks for moving this here.

I just realized. The actual syntax for using port numbers is `!PORT`. Like so:

    \\sshfs\<USER_NAME>@<DOMAIN>!<PORT>

**Comment by tgckpg** [comment]
Just tried, still prompting  for passwords.
// Renaming issue

**Comment by billziss-gh** [comment]
Hmm... I think I do have an SSH server that requires a port number. I will be able to check this in a couple of hours.

**Comment by billziss-gh** [comment]
Hi, @tgckpg. Unfortunately my SSH server (that requires a port number) is no longer online. So I cannot troubleshoot directly.

What is the exact error message you are getting? Can you ping the server from the command prompt on the same machine? Can you SSH into it from the same machine (e.g. using Cygwin)?

**Comment by tgckpg** [comment]
Was going to use tcpdump to dump those connection and realized I've been connecting to a wrong port the whole time.

Sorry this is my fault and not a bug.

**Comment by billziss-gh** [comment]
No problem. Thanks for letting me know.

**Comment by sindzicat** [comment]
Hello!
I have the same problem. The port is correct. I try to connect to my home ubuntu server. What there should be installed on ubuntu server to make connection work?

**Comment by sindzicat** [comment]
I solved my problems. I was need to install [sshfs-win](https://github.com/billziss-gh/sshfs-win/releases/latest) for that.

**Comment by HenrikBach1** [comment]
Hi,

It is not that obvious that one should use above syntax to connect remotely via a port. How come? Why not use <user>@<host>:<port> syntax as properly everybody which has used ssh will do. 

I did that in very long time, but in vein as reported. I suspected my ssh installation wasn't installed somehow successfully. In the end, I searched in these closed issues and found the method to connect.

/Henrik
